nipype/interfaces/io.py

```diff
@@ -2863,3 +2864,52 @@ def _list_outputs(self):
 
     def _add_output_traits(self, base):
         return add_traits(base, list(self.inputs.output_query.keys()))
+
+
+class ExportFileInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='Input file name')
+    out_file = File(mandatory=True, desc='Output file name')
+    check_extension = traits.Bool(True, desc='Ensure that the input and output file extensions match')
+    clobber = traits.Bool(desc='Permit overwriting existing files')
+
+
+class ExportFileOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='Output file name')
+
+
+class ExportFile(SimpleInterface):
+    """Copy in_file to out_file.
+
+    This interface copies an input file to a named output file.
+    This is useful to save files to a specified location
+    (as opposed to using DataSink).
+
+    Examples
+    --------
+
+    A trivial example that copies temporary_file.nii.gz
+    to sub1_out.nii.gz. (A more realistic example would set
+    in_file as the output of another Node.)
```

Comment:
This feels like an unnecessary qualifier. I think you could probably just skip this text, unless you want to have multiple tests showing off `check_extension` and `clobber`. If so, you could say "By default, file extensions need to match, and the output file should not exist:" and something similar above the other examples.